webapp/app/css/treeherder.css

```diff
@@ -418,10 +418,30 @@ th-watched-repo {
     display: flex;
     display: -webkit-flex;
     -webkit-flex-flow: row wrap;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.result-set-left {
+    display: flex;
+    display: -webkit-flex;
+    -webkit-flex-flow: row wrap;
     flex-flow: row wrap;
+    flex: auto;
     align-items: center;
 }
 
+.result-set-progress {
+    color: #6f6d70;
+    font-style: italic;
+    margin-left: 10px;
```

Comment:
I think we could move this further to the right - ie on my window size the "In progress" counts appear really close the the revision list, rather than above the job counts. Perhaps we should be aligning relative to the buttons on the right hand side instead? :-)